fe-public/src/layouts/Layout.astro

```diff
@@ -64,12 +64,12 @@ const { title } = Astro.props;
       g.async=true; g.src='https://cdn.matomo.cloud/danielalvesworld.matomo.cloud/container_C1J2mKYr.js'; s.parentNode.insertBefore(g,s);
     })();
   </script>
-  <!-- End Matomo Tag Manager -->
+  <!-- End Matomo Tag Manager -->  <!-- Maybe import footer in layout instead -->
```

Comment:
I think we can keep it for the future, since for now we just have the landing page primarily